staging/src/k8s.io/client-go/rest/config.go

```diff
@@ -476,15 +476,26 @@ func DefaultKubernetesUserAgent() string {
 		adjustCommit(version.Get().GitCommit))
 }
 
+func getPath(path string) string {
```

Comment:
Maybe `getServiceAccountPath` as `getPath` isn't very descriptive. (and add a comment on the need for it)

Reply:
Well, I guess you pass in any path here so it could be used for more than the service account. Do we envision this being used for anything other than the service account path though? If not I'd just throw the consts for the service account paths in here instead.

Reply:
I had similiar thoughts. I am testing this more now, to see if we need this in other locations.  It maybe needed here as well https://github.com/kubernetes/kubernetes/blob/e007ca872a5cdecacb3c0b9402beaba3714ec60a/staging/src/k8s.io/client-go/rest/config.go#L544